src/haddock/modules/refinement/emref/__init__.py

```diff
@@ -93,11 +94,12 @@ def run(self, **params):
         jobs = []
 
         # Get the models generated in previous step
-        models_to_refine = [
-            p
-            for p in self.previous_io.output
-            if p.file_type == Format.PDB
-            ]
+        models_to_refine = load_from_previous(self.previous_io.output,
```

Comment:
Does this work? `check_balance` is not defined in the function definition.

Reply:
ops forgot to delete it